examples/tts/aligner_g2p.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Maybe rename the file to smth more descirptive, i.e. "aligner_heteronym_disambiguation.py"?
Also "& AFFILIATES" is missing in the header

Reply:
Makes sense, updated.